src/script/cpp_api/s_security.cpp

```diff
@@ -255,6 +256,7 @@ void ScriptApiSecurity::initializeSecurityClient()
 	};
 	static const char *debug_whitelist[] = {
 		"getinfo",
+		"traceback",
```

Comment:
it's need for the error handler since minetest needs to support lua 5.1 and afaik that version of lua has way to get a traceback using the C API. I could store it in an RIDX if that would be better.